README.md

```diff
@@ -80,32 +80,61 @@ metric exporters, service monitors, and custom dashboards.
 See the [logging README](logging/README.md) to deploy the logging components,
 including Fluent Bit, Elasticsearch, and Kibana.
 
-## Miscellaneous
+## Customization
 
-### Customization
+The process of customizing the monitoring and logging deployments consists of: 
+- creating the location for your local customization files
+- specifying customization variables and parameters in the customization files
+- using the `USER_DIR` environment variable to specify the location of the customization files
 
-The components that are deployed are highly customizable. The recommended
-approach for customization is to set a `USER_DIR` environment variable. `USER_DIR`
-points to a directory containing user-modified customization files which can exist
-outside of your repository. You can use customization files to check in
-environment-specific customizations into a separate Git repository, for example.
+### Create the Location for Customization Files
```

Comment:
I think this is a good place to mention the generic base sample. The point of that sample is to provide a starting point for a USER_DIR with the proper structure and files that include usage comments.

Reply:
I agree we should highlight using generic-base as the "template" users should use for creating their own USER_DIR.  But we should probably be very explicit that this is the case _only_ if none of the other samples are _more appropriate_ for their situation.  I'd like to avoid forcing customers to reconcile multiple samples together if they don't have to.    On the other hand, maybe there is an advantage in telling folks to _always_ create their USER_DIR from generic-base and then documenting the other samples as starting from that known starting state.

Reply:
Using generic-base as a start is now documented as a best practice, but using another sample as a base is documented as an alternative approach